index.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+require 'yaml'
+require 'faker'
+require 'date'
+
+require_relative 'entities/author'
+require_relative 'entities/book'
+require_relative 'entities/reader'
+require_relative 'entities/order'
+
+require_relative 'generator_fake_data'
+require_relative 'create_database'
+require_relative 'statistic'
+require_relative 'validation/errors'
+
+# main class
+class Library
+  include CreateDatabase
+  include Statistics
+
+  def initialize
+    if File.exist?('seeds.yaml')
+      load_data
+    else
+      generate_data
+      save_data
+    end
+  end
+
+  def delete_file
+    delete_data
+  end
+
+  def statistics
```

Comment:
i think, it will be better to call this methods in index.rb and remove `statistics` method

Reply:
corrected